pkg/kubelet/dockershim/BUILD

```diff
@@ -53,9 +53,9 @@ go_library(
         "//pkg/security/apparmor:go_default_library",
         "//pkg/util/exec:go_default_library",
         "//pkg/util/hash:go_default_library",
-        "//pkg/util/term:go_default_library",
         "//vendor/github.com/blang/semver:go_default_library",
         "//vendor/github.com/docker/docker/pkg/jsonmessage:go_default_library",
+        "//vendor/github.com/docker/docker/pkg/term:go_default_library",
```

Comment:
I think it's ok to vendor 3rd party code, but not ok for kubectl to depend on main k8s repo.